.github/workflows/auto-add-review-lebel.yml

```
@@ -0,0 +1,23 @@
+name: Auto Add Review Label
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  add_review_label:
+    if: github.event.pull_request.draft != true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```
せっかくなので v3 にしませんか (他の箇所も同様)